main.go

```diff
@@ -45,11 +45,15 @@ var (
 	argElasticURL = ""
 	argUsername   = ""
 	argPassword   = ""
+	argRepoAccessKey = ""
+	argRepoSecretKey = ""
+	argRepoRegion = "eu-west-1"
```

Comment:
I think we shouldn't have a default here, it causes issues for folks who don't set but operate in different regions.

Reply:
Right, I have removed the default here.